src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -0,0 +1,152 @@
+//#define ENERGY_PRESERVING_MONOCHROME
+
+float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+
+	if ( decayExponent > 0.0 ) {
+
+	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+
+	}
+
+	return 1.0;
+
+}
+
+
+void BRDF_Lambert( const in IncidentLight incidentLight, const in GeometricContext geometryContext, const in vec3 diffuseColor, inout ReflectedLight reflectedLight ) {
+
+	float lambertianReflectance = saturate( dot( geometryContext.normal, incidentLight.direction ) );
+
+	#if defined( ENERGY_PRESERVING_MONOCHROME ) || defined( ENERGY_PRESERVING_RGB )
+
+		lambertianReflectance *= RECIPROCAL_PI;
```

Comment:
I can cite the PBRT renderer: https://github.com/mmp/pbrt-v3/blob/master/src/core/reflection.cpp#L145

But seriously, just do the integral over the hemisphere, if you do not do this divide, you get way more light out than what is coming in.